src/SFA.DAS.CloudService/SFA.DAS.EAS.WebContent/diagnostics.wadcfgx

```diff
@@ -43,5 +43,5 @@
   <PrivateConfig>
     <StorageAccount name="" key="" endpoint="" />
   </PrivateConfig>
-  <IsEnabled>true</IsEnabled>
+  <IsEnabled>false</IsEnabled>
```

Comment:
what is the motivation behind this change?

Reply:
Dev Ops removed a load of the diagnostics config so the app wouldn't run locally without making that change.